suse_migration_services/exceptions.py

```diff
@@ -86,3 +86,9 @@ class DistMigrationZypperException(DistMigrationException):
     """
     Exception raised if the zypper migration call has failed
     """
+
+
+class DistMigrationGrubException(DistMigrationException):
```

Comment:
It's all about the config, we are not installing grub. Thus I think we should be specific e.g
DistMigrationGrubConfigException